EIPS/eip-3722.md

```diff
@@ -163,17 +175,16 @@ While there would have been some gas savings if we decided against emitting anyt
 
```

Comment:
There's a mandatory "Test Cases" section that should go here (even though this is a ridiculously simple contract). You can link to the test case in the repo in the Reference Implementation.

Reply:
Test Cases and Reference Implementation sections are both optional for ERCs.  Test Cases is required only for consensus changes (Core EIPs).

Reply:
oh whoops, sorry about that